ui/src/index.js

```diff
@@ -6,8 +6,36 @@ import '@patternfly/react-core/dist/styles/base.css';
 import './index.css';
 import App from './components/App';
 import configureStore from './configureStore';
+import { SortByDirection, sortable } from '@patternfly/react-table';
 
-const preloadedState = {};
+const preloadedState = {
```

Comment:
@damienomurchu is this just for testing?

Reply:
@more of an experiment really. I see we're preloading data in the reducer index.js and wondered if that was the best place to do it, so when refactoring toyed with putting it in here. It wasn't without problems, so will most likely remove it & if its deemed task-worthy we can do it an another PR at at later stage

Reply:
removing this for now, so resolving this